mycroft/skills/fallback_skill.py

```diff
@@ -122,8 +130,16 @@ def wrapper(*args, **kwargs):
                 return True
             return False
 
-        self.instance_fallback_handlers.append(wrapper)
-        self._register_fallback(wrapper, priority)
+        self.instance_fallback_handlers.append(handler)
+        self._register_fallback(handler, wrapper, priority)
+
+    @classmethod
+    def _remove_registered_handler(cls, wrapper_to_del):
+        for priority, handler in list(cls.fallback_handlers.items()):
+            if handler == wrapper_to_del:
+                del cls.fallback_handlers[priority]
+        else:
+            LOG.warning('No fallback matching {}'.format(wrapper_to_del))
```

Comment:
I'm hitting this warning when I reload fallback-query  by touching `__init__.py`. 
Should the `handle_question` fallback be getting removed?

Reply:
That doesn't sound correct, let me check...

Reply:
Ok, found the mistake. Since the code above doesn't break to exit the for loop (to be able to handle multiple registered wrappers) the warning will always occur. I'll fix that.